backend/question-service/server/app/.env.example

```diff
@@ -1,10 +1,8 @@
+MONGO_CONNECTION_STRING=mongodb://localhost:27017
```

Comment:
same thing as above, can name it as
`mongodb://questions-service-db:27017`

cos our convention is to provide the docker compose env vars in each app .env file instead of specifying in the docker compose like u did

Reply:
Resolved in f38c245482f8f1e717f5d1f41dc82882b7ce0908:
- Changed to use docker-compose env var in `.env.development`
- Removed environment variable from docker-compose